src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -150,8 +150,7 @@ std::vector<std::vector<int>> get_streams_info_table(const int input_streams,
          ((latencyThreadingMode == Config::LatencyThreadingMode::PER_PLATFORM) || (proc_type_table.size() == 1))) ||
         ((input_streams_changed == true) && (input_streams == 1))) {
         n_streams = 1;
-        if ((proc_type_table.size() == 1) && (model_prefer_threads > 0) &&
-            ((input_threads == 0) || (input_threads == 0))) {
+        if ((proc_type_table.size() == 1) && (model_prefer_threads > 0) && ((input_threads == 0))) {
```

Comment:
The double brackets here can be changed to single brackets.

Reply:
updated